Sources/LiveViewNative/Image Modifiers/InterpolationModifier.swift

```diff
@@ -15,11 +15,11 @@ import SwiftUI
 /// ```html
 /// <Image
 ///   name="dot_green"
-///   modifiers={@native |> resizable() |> interpolation(interpolation: :none)}
+///   modifiers={resizable([]) |> interpolation(:none)}
 /// />
 /// <Image
 ///   name="dot_green"
-///   modifiers={@native |> resizable() |> interpolation(interpolation: :medium)}
+///   modifiers={resizable([]) |> interpolation(:medium)}
```

Comment:
Why do we have an empty list being sent?

Reply:
Hmm, there might still be a bug with 0-arity modifiers in `live_view_native` but I thought we fixed this. I'll look into it and fix and/or update the docs in a subsequent PR.